app/controllers/obituaries/covid_controller.rb

```diff
@@ -1,5 +1,7 @@
 class Obituaries::CovidController < ApplicationController
   def index
-    @obituaries = Obituary.where(covid: true) 
+    @obituaries_api = ObituarySearch.new.search_covid_obituaries
```

Comment:
The way I have this now both of these will be displayed on the screen, with the ones in our database on top and the ones from the api on the bottom

Reply:
excellent!